common/ethics/00_ethics.txt

```diff
@@ -2157,7 +2157,7 @@ ethic_fanatic_materialist = {
 	
 	country_modifier = {
 		planet_jobs_upkeep_mult = -0.10
-		country_physics_tech_research_speed = 0.20
+		planet_max_districts_add = 2
```

Comment:
I live the move of districts onto materialism. Perhaps all the research bonuses could be replaced with other bonuses on ethics.

Reply:
Do you still want to replace all the research bonuses as you mentioned in the conversation you opened and resolved? I think militarist engineering bonus could be replaced with a naval capacity bump. Not sure yet on xenophile society bonus - will give it some thought

Reply:
Trust growth perhaps? Without meaningful internal politics it is often difficult to find bonuses that don't encroach too much on diplomat

Reply:
+% Naval Capacity on Militarist is fine.

Perhaps Xenophile could get a +Habitability and bonuses to Habitability from some other areas can be removed.

Reply:
That's a great idea actually. Lets you maintain worlds with species of diff climate prefs w/o needing gaias, ecus, or habitats. Right now of course all of those are easy to get and cheap but the forthcoming strat rework will hopefully make the bonus more meaningful.